src/OpenFeature.Contrib.Hooks.Otel/OtelHook.cs

```diff
@@ -1,18 +1,29 @@
-﻿using OpenFeature;
-using OpenFeature.Model;
+﻿using OpenFeature.Model;
 using System.Threading.Tasks;
 using System.Collections.Generic;
-using OpenTelemetry.Context.Propagation;
-using OpenTelemetry.Trace;
+using System.Diagnostics.CodeAnalysis;
+using System;
 
 namespace OpenFeature.Contrib.Hooks.Otel
 
 {
     /// <summary>
     /// Stub.
     /// </summary>
+    [ExcludeFromCodeCoverage]
+    [Obsolete("This class is obsolete and will be removed in a future version. Please use TracingHook instead.")]
     public class OtelHook : Hook
     {
+        private readonly TracingHook _tracingHook;
+
+        /// <summary>
+        /// Stub.
```

Comment:
Can you remove this?

Reply:
Fixed with https://github.com/open-feature/dotnet-sdk-contrib/pull/116/commits/00b17642ec7d4f8004ceeab48719de780544eabf